app/server.rb

```diff
@@ -0,0 +1,100 @@
+require 'sinatra/base'
+require 'data_mapper'
+require 'rack-flash'
+
+class Chitter < Sinatra::Base
+
+  env = ENV['RACK_ENV'] || 'development'
+
+  enable :sessions
+  use Rack::Flash
+  use Rack::MethodOverride
+
+  set :views, Proc.new {File.join(root, "..", "views")}
+  set :session_secret, 'secret session'
+
+
+  DataMapper.setup(:default, "postgres://localhost/chitter_#{env}")
+
+  require_relative './lib/cheet'
+  require_relative './lib/user'
+
+  DataMapper.finalize
+  DataMapper.auto_upgrade!
+
+
+  get '/' do
```

Comment:
You could split all these routes into a controller file to clean up your server file